setup.py

```diff
@@ -17,14 +22,14 @@ def get_version():
     """
     locals_dict = {}
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
+    with open(os.path.join(__curdir__, 'nilearn', 'version.py')) as fp:
```

Comment:
Here (and below) allows `setup.py` to be called from a different working directory than the `nilearn` directory.  Not sure if it's every needed by `pypi`, but it did help me with testing... :)